Buy/Generated/Storefront/TokenizedPaymentInputV2.swift

```diff
@@ -54,29 +51,32 @@ extension Storefront {
 		/// Public Hash Key used for AndroidPay payments only. 
 		open var identifier: Input<String>
 
+		/// The type of payment token. 
+		open var type: String
+
 		/// Creates the input object.
 		///
 		/// - parameters:
 		///     - paymentAmount: The amount and currency of the payment.
 		///     - idempotencyKey: A unique client generated key used to avoid duplicate charges. When a duplicate payment is found, the original is returned instead of creating a new one.
 		///     - billingAddress: The billing address for the payment.
-		///     - type: The type of payment token.
 		///     - paymentData: A simple string or JSON containing the required payment data for the tokenized payment.
 		///     - test: Executes the payment in test mode if possible. Defaults to `false`.
 		///     - identifier: Public Hash Key used for AndroidPay payments only.
+		///     - type: The type of payment token.
 		///
-		public static func create(paymentAmount: MoneyInput, idempotencyKey: String, billingAddress: MailingAddressInput, type: String, paymentData: String, test: Input<Bool> = .undefined, identifier: Input<String> = .undefined) -> TokenizedPaymentInputV2 {
-			return TokenizedPaymentInputV2(paymentAmount: paymentAmount, idempotencyKey: idempotencyKey, billingAddress: billingAddress, type: type, paymentData: paymentData, test: test, identifier: identifier)
+		public static func create(paymentAmount: MoneyInput, idempotencyKey: String, billingAddress: MailingAddressInput, paymentData: String, type: String, test: Input<Bool> = .undefined, identifier: Input<String> = .undefined) -> TokenizedPaymentInputV2 {
+			return TokenizedPaymentInputV2(paymentAmount: paymentAmount, idempotencyKey: idempotencyKey, billingAddress: billingAddress, paymentData: paymentData, type: type, test: test, identifier: identifier)
 		}
 
-		private init(paymentAmount: MoneyInput, idempotencyKey: String, billingAddress: MailingAddressInput, type: String, paymentData: String, test: Input<Bool> = .undefined, identifier: Input<String> = .undefined) {
+		private init(paymentAmount: MoneyInput, idempotencyKey: String, billingAddress: MailingAddressInput, paymentData: String, type: String, test: Input<Bool> = .undefined, identifier: Input<String> = .undefined) {
```

Comment:
Changes like this should be closely monitored and discarded. Ordering of function parameters in Swift is deterministic. Any changes to this order will incur unnecessary breaking changes within existing code bases.

Reply:
@dbart01 Good point. Although I don't blame the generator for the re ordering. The re ordering was the effect of an actual API change. I'll discard this change here